sys/net/gnrc/network_layer/ipv6/netif/gnrc_ipv6_netif.c

```diff
@@ -776,6 +777,10 @@ void gnrc_ipv6_netif_init_by_dev(void)
 
             DEBUG("ipv6 netif: Set 6LoWPAN flag\n");
             ipv6_ifs[i].flags |= GNRC_IPV6_NETIF_FLAGS_SIXLOWPAN;
+#ifdef MODULE_GNRC_SIXLOWPAN_ND_ROUTER
+            DEBUG("ipv6 netif: Set router flag\n");
+            ipv6_ifs[i].flags |= GNRC_IPV6_NETIF_FLAGS_ROUTER;
```

Comment:
Will be set (with correct send-out behavior) in https://github.com/RIOT-OS/RIOT/pull/3813/files#diff-f9bf85cb337616822921e4230ede1572R830